gradle/libs.versions.toml

```diff
@@ -0,0 +1,264 @@
+prometheusClientBom = { module = "io.prometheus:simpleclient_bom", version.ref = "prometheus" }
+prometheusHotspot = { module = "io.prometheus:simpleclient_hotspot", version.ref = "prometheus" }
+prometheusHttpserver = { module = "io.prometheus:simpleclient_httpserver", version.ref = "prometheus" }
+protoGoogleCommon = { module = "com.google.api.grpc:proto-google-common-protos", version.ref = "protoGoogleCommon" }
+protobufJava = { module = "com.google.protobuf:protobuf-java", version.ref = "protobufJava" }
+protoc = { module = "com.google.protobuf:protoc", version.ref = "protoc" }
+retrofit = { module = "com.squareup.retrofit2:retrofit", version.ref = "retrofit" }
+retrofitMock = { module = "com.squareup.retrofit2:retrofit-mock", version.ref = "retrofit" }
+retrofitMoshi = { module = "com.squareup.retrofit2:converter-moshi", version.ref = "retrofit" }
+retrofitProtobuf = { module = "com.squareup.retrofit2:converter-protobuf", version.ref = "retrofit" }
+retrofitScalars = { module = "com.squareup.retrofit2:converter-scalars", version.ref = "retrofit" }
+retrofitWire = { module = "com.squareup.retrofit2:converter-wire", version.ref = "retrofit" }
+servletApi = { module = "javax.servlet:javax.servlet-api", version.ref = "servletApi" }
+slf4jApi = { module = "org.slf4j:slf4j-api", version.ref = "slf4j" }
+tempest2Testing = { module = "app.cash.tempest:tempest2-testing", version.ref = "tempest" }
+tempest2TestingDocker = { module = "app.cash.tempest:tempest2-testing-docker", version.ref = "tempest" }
+tempest2TestingInternal = { module = "app.cash.tempest:tempest2-testing-internal", version.ref = "tempest" }
+tempest2TestingJvm = { module = "app.cash.tempest:tempest2-testing-jvm", version.ref = "tempest" }
+tempestTesting = { module = "app.cash.tempest:tempest-testing", version.ref = "tempest" }
+tempestTestingDocker = { module = "app.cash.tempest:tempest-testing-docker", version.ref = "tempest" }
+tempestTestingInternal = { module = "app.cash.tempest:tempest-testing-internal", version.ref = "tempest" }
+tempestTestingJvm = { module = "app.cash.tempest:tempest-testing-jvm", version.ref = "tempest" }
+tink = { module = "com.google.crypto.tink:tink", version.ref = "tink" }
+tinkAwskms = { module = "com.google.crypto.tink:tink-awskms", version.ref = "tink" }
+tinkGcpkms = { module = "com.google.crypto.tink:tink-gcpkms", version.ref = "tink" }
+wireBom = { module = "com.squareup.wire:wire-bom", version.ref = "wire" }
+wireGrpcClient = { module = "com.squareup.wire:wire-grpc-client", version.ref = "wire" }
+wireMoshiAdapter = { module = "com.squareup.wire:wire-moshi-adapter", version.ref = "wire" }
+wireReflector = { module = "com.squareup.wire:wire-reflector", version.ref = "wire" }
+wireRuntime = { module = "com.squareup.wire:wire-runtime", version.ref = "wire" }
+wireSchema = { module = "com.squareup.wire:wire-schema", version.ref = "wire" }
+wispAwsEnvironment = { module = "app.cash.wisp:wisp-aws-environment", version.ref = "wisp" }
```

Comment:
unsure if i'm supposed to specify `version.ref = "wisp"` for all of these 🤔 or if there's a cleaner way

Reply:
I'd reference the internal Android repo, probably the biggest, most complex Gradle lib.versions.toml I've seen. Guessing if there's a better way, it is there.